ts/components/messages/MessageCTABar.tsx

```diff
@@ -154,6 +155,14 @@ class MessageCTABar extends React.PureComponent<Props, State> {
     }
   }
 
+  private openAppSettings = () => {
```

Comment:
The same method is also declared in another screen please create a util file.

Reply:
@fpersico I didn't do it because I check other _openAppSettings_ functions and each one uses a different string parameter (iOS) or use different open settings functions (Android)

### biometric screen
```javascript
private openAppSettings = () => {
    if (Platform.OS === "ios") {
      Linking.openURL("App-prefs:root=General").catch(_ => undefined);
    } else {
      AndroidOpenSettings.securitySettings();
    }
  };
```

### scan qr screen
(don't why in scan qr ask to open notification settings part)
```javascript
  private openAppSettings = () => {
    if (Platform.OS === "ios") {
      Linking.openURL("app-settings://notification/IO").catch(_ => undefined);
    } else {
      AndroidOpenSettings.appDetailsSettings();
    }
  };
```

btw I uniformed all these function in 2:
**openAppSettings**
**openAppSecuritySettings**

You can find them in last commit